packages/dev/test-utils/src/renderOverride.js

```diff
@@ -34,3 +37,7 @@ function customRender(ui, options) {
 
 // override render method with
 export {customRender as render};
+
+export function renderv3(ui, options, providerProps) {
```

Comment:
just confirming: this isn't the public test utils package right?

Reply:
correct, this is private

Reply:
https://github.com/adobe/react-spectrum/blob/main/packages/dev/test-utils/package.json#L4